src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -245,6 +245,13 @@
       "The next card you play this turn is played an additional #b%1$s times."
     ]
   },
+  "stsjorbsmod:PossessionPower": {
+    "NAME": "Possessed",
+    "DESCRIPTIONS": [
+      "This turn instead of attacking, this creature will attack a random ally for #b%1$s damage.",
```

Comment:
Thinking about it a bit more, enemy intents aren't always just attacking. Perhaps "This turn, instead of taking its normal turn,..."? I dunno, that feels clunky. Ideally, we could check the intent and swap "attacking" with "buffing", "debuffing", or whatever is good text for "???". These texts would also be localized.

I would prefer these singular and plural be consistent.